doc/conf.py

```diff
@@ -57,14 +61,71 @@
 
 intersphinx_mapping = haddock_mapping
 
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['.sphinx/_templates']
+html_static_path = ['.sphinx/_static']
+
+source_suffix = {
+    '.rst': 'restructuredtext',
+    '.md': 'markdown',
+}
+
+intersphinx_mapping = {
+    "commandsv1": (
```

Comment:
Why are we linking to the docs for FIRST? Also this overrides the mapping set earlier

Reply:
Removed.